src/chrome/content/rules/RB-Media-Group.de.xml

```diff
@@ -0,0 +1,83 @@
+<!--
+	Mismatched:
+		- mail
+		- nas
+		- rbbackups1
+
+	No route to host:
+		- accounting
+		- apc3
+		- dns1
+		- server
+
+	SSL error:
+		- sitemap
+
+	Incomplete certificate chain:
+		- vmware
+
+	Connection refused:
+		- apc1
+		- apc4
+		- ats1
+		- ats2
+		- ats3
+		- backups1
+		- dns2
+		- dns3
+		- ns1
+		- ns2
+		- ns3
+		- ns4
+		- ns5
+		- ns6
+		- ns7
+
+	Self-signed:
+		- billing
+		- cms
+		- crm
+		- dev
+		- ftp
+		- imap
+		- m
+		- pop3
+		- smtp
+		- thomastest
+
+	Connection timed out:
```

Comment:
I got a mismatching certificate or connection refused on some of them, please double check.